cadcutils/cadcutils/net/ws.py

```diff
@@ -767,6 +763,21 @@ def _resolve_destination_file(dest, src_md5, default_file_name):
         else:
             return final_dest, final_dest
 
+    def compute_file_md5(file_path):
```

Comment:
This duplicates the method `_get_md5sum` in `storageinv.py`, and also at line 897 of this file. Not for now, but maybe a future consolidation?

Reply:
`_get_md5sum` seemed to be a leftover after some rework. Removed it since it was supposed to be an internal method.
Re-worked the `compute_file_md5` function to accommodate the functionality required at line 897. Also, scaled down the size of the buffer in that function.
Please let me know if this is acceptable.